BinarySequense/BinarySequense/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BinarySequense
+{
+    class Program
+    {
+        static void Main()
+        {
+            var text = "";
+           
+            try
+            {
+                using(StreamReader sr = new StreamReader("binary_sequense.txt"))
+                {
+                   text = sr.ReadToEnd();
+                }
+            }
+            catch { }
+
+            var listByte = ParserText(text);
+            var sortList = new int[listByte.Count];
+            listByte.CopyTo(sortList);
+            sortList.ToList().Sort();
+            WriteSpecialNumber(sortList);
+            WriteMaxSequense(listByte);
+            Console.ReadLine();
+        }
+
+        
+        private static void WriteMaxSequense(List<int> listByte)
+        {
+            var currentElement = 0;
+            var currentList = new List<int>();
+            var result = new List<int>();
+            foreach(var element in listByte)
+            {
+                if(currentElement < element)
+                    currentList.Add(element);
+                else
+                {
+                    if (result.Count < currentList.Count)
+                        result = currentList;
+                    currentList = new List<int>();
+                }
+                currentElement = element;
+            }
+            if (result.Count == 0)
+                result = currentList;
+           
+            result.Sort();
+           
+            foreach (var element in result)
+                Console.Write(element + " ");
+        }
+
+       
+        private static void WriteSpecialNumber(int[] sortList)
+        {
+            var lastElemenet = sortList[0];
+            Console.Write(lastElemenet + " ");
+            foreach(var element in sortList)
+                if(lastElemenet != element)
+                {
+                    Console.Write(element + " ");
+                    lastElemenet = element;
+                }
+            Console.WriteLine();
```

Comment:
Саму последовательность выводить не нужно.
```suggestion
            foreach(var element in sortList)
                if(lastElemenet != element)
                {
                    lastElemenet = element;
                }
```